src/nemos/glm.py

```diff
@@ -1376,6 +1453,26 @@ def fit(
         - If the mask is a `FeaturePytree`, then `"feature_name"` is a predictor of neuron `j` if
         `feature_mask["feature_name"][j] == 1`.
 
+        Examples
```

Comment:
I find this example confusing. maybe add a couple of comments explaining what weights are and what they're used for here? feature_mask is fine, because it's explained elsewhere in the docstring